src/authentication.ts

```
@@ -136,15 +136,13 @@ export default class GitpodAuthenticationProvider extends Disposable implements
 			const scopesStr = sortedScopes.join(' ');
 
 			let userInfo: { id: string; accountName: string } | undefined;
-			if (!session.account) {
-				try {
-					userInfo = await this._gitpodServer.getUserInfo(session.accessToken);
-					this._logger.info(`Verified session with the following scopes: ${scopesStr}`);
-				} catch (e) {
-					// Remove sessions that return unauthorized response
-					if (e.message === 'Unauthorized') {
-						return undefined;
-					}
+			try {
+				userInfo = await this._gitpodServer.getUserInfo(session.accessToken);
+				this._logger.info(`Verified session with the following scopes: ${scopesStr}`);
+			} catch (e) {
+				// Remove sessions that return unauthorized response
```

It seem we are swallowing errors here, i think we should log them at least